mars/metric/api.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Dict, Any, Optional, Tuple
+from .backends.console import console_metric
```

Comment:
Basically, we separate imports with blank lines.

```
import logging  # internal

import numpy  # third-party

from .backends import xxx  # mars itself
```

Reply:
Ok. Thanks a lot.